package.json

```diff
@@ -98,6 +98,16 @@
       }
     }
   },
+  "detox": {
+    "configurations": {
+      "ios.sim.debug": {
+        "binaryPath": "playground/ios/DerivedData/playground/Build/Products/Debug-iphonesimulator/playground.app",
+        "build": "xcodebuild -project playground/ios/playground.xcodeproj -scheme playground -configuration Debug -sdk iphonesimulator RCT_NO_LAUNCH_PACKAGER=true -derivedDataPath playground/ios/DerivedData/playground ONLY_ACTIVE_ARCH=YES",
```

Comment:
should be `RCT_NO_LAUNCH_PACKAGER=true xcodebuild build -project playground/ios/playground.xcodeproj -scheme playground -configuration Debug -sdk iphonesimulator -derivedDataPath playground/ios/DerivedData/playground ONLY_ACTIVE_ARCH=YES -quiet`